src/hawkbit-client.c

```diff
@@ -615,13 +615,13 @@ gboolean process_deployment(JsonNode *req_root, GError **error)
                 goto proc_error;
         }
 
-        g_message("New software ready for download. (Name: %s, Version: %s, Size: %ld, URL: %s)",
```

Comment:
What about simply using the G_GINT64_FORMAT define here?

    g_message("New software ready for download. (Name: %s, Version: %s, Size: %"G_GINT64_FORMAT ", URL: %s)"